pages/index.vue

```diff
@@ -16,6 +22,74 @@ onMounted(() => {
   <div>
     <section>
       <div class="content">
+        <template v-if="featuredArticles">
+          <div class="grid mb-6">
+            <div class="col-12 xl:col-8">
+              <v-card
+                class="mod-vertical mod-featured mod-large"
+                :image="useImageUrl(featuredArticles[0].listingImage)"
+                :sizes="[1]"
+                :title="featuredArticles[0].title"
+                :titleLink="featuredArticles[0].link"
+                :maxWidth="featuredArticles[0].image.width"
+                :maxHeight="featuredArticles[0].image.height"
+                :tags="[
+                  {
+                    name: featuredArticles[0].section.name,
+                    slug: featuredArticles[0].section.slug,
+                  },
+                ]"
+              >
+                <p class="desc">
+                  {{ featuredArticles[0].description }}
+                </p>
+                <div class="article-metadata">
+                  <span>
+                    <v-byline :authors="featuredArticles[0].authors" />
+                  </span>
+                  <span>comments</span>
+                </div>
+              </v-card>
+            </div>
+            <div class="col-12 xl:col-4">
+              <hr class="black mb-3" />
+              <div class="mb-3">LATEST</div>
+              <div
+                v-for="article in featuredArticles.slice(
+                  1,
+                  featuredArticles.length
+                )"
+                :key="article.uuid"
+              >
```

Comment:
I have an idea to make these cards less crazy but it can wait until we have more of them set up.